app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,24 @@
-<h2>Log in</h2>
-
+<div class="register">
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
+  <div class="lines">
+    <%= f.label :email %>
     <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
 
-  <div class="field">
-    <%= f.label :password %><br />
+  <div class="lines">
+    <%= f.label :password %>
     <%= f.password_field :password, autocomplete: "current-password" %>
   </div>
 
   <% if devise_mapping.rememberable? %>
-    <div class="field">
+    <div class="lines">
       <%= f.check_box :remember_me %>
       <%= f.label :remember_me %>
     </div>
   <% end %>
-
-  <div class="actions">
-    <%= f.submit "Log in" %>
+  <div class="sign_up_button_container">
+    <%= f.submit "Log in", class: "sign_up_button" %>
   </div>
```

Comment:
@HFG43 I kindly suggest you add custom styles for the button and give a little more space between the form fields and the button.